tests/test_examples.py

```diff
@@ -326,6 +326,8 @@ def to_test(
             return True
         elif "huggyllama" in model_name and IS_GAUDI2 and deepspeed:
             return True
+        elif "gemma" in model_name and IS_GAUDI2:
```

Comment:
I updated this elif because the current flow always skip this  check

Reply:
Yes you did well! But we also need to keep the former rule otherwise the deepspeed test is not registered.

Reply:
ok thank you!